src/index.js

```diff
@@ -1,35 +1,163 @@
+  addDataToLocal(arrayOfTasks);
+}
+
+function addElementsToPageFrom(arrayOfTasks) {
+  // empty task div if has any data
+  tasksDiv.innerHTML = ' ';
+
+  // looping arryof tasks
+
+  arrayOfTasks.forEach((task) => {
+    // create main div
+    const div = document.createElement('div');
+    div.className = 'task';
+
+    // check if task id done
+    if (task.completed) {
+      div.className = 'task done';
+    }
+    div.setAttribute('data-id', task.id);
+    // div.appendChild(document.createTextNode(task.title));
 
-const displayTasks = (lists) => {
-  lists.forEach((list) => {
-    const html = ` <div class"task-div"> <input type="checkbox" name="task" value="task">
-       <label class="list-item" for="task${list.index}"> ${list.description}</label><br>
-      </div>`;
+    // create checkbox
+    const checkbox = document.createElement('input');
+    checkbox.setAttribute('type', 'checkbox');
+    checkbox.className = 'checked';
+    checkbox.appendChild(document.createTextNode(''));
+    div.appendChild(checkbox);
 
-    listContainer.insertAdjacentHTML('beforeend', html);
+    // create input description
+    const desc = document.createElement('input');
+    desc.setAttribute('value', task.title);
+    desc.className = 'desc';
+    div.appendChild(desc);
+
+    // create delete span
+    const span = document.createElement('span');
+    span.className = 'del';
+    span.appendChild(document.createTextNode('Delete'));
+    // append button to div
+    div.appendChild(span);
+
+    desc.addEventListener('change', (e) => {
+      task.title = e.target.value;
+      // eslint-disable-next-line no-use-before-define
+      addDataToLocal(arrayOfTasks);
+    });
+
+    // add div to container
+    tasksDiv.appendChild(div);
   });
-};
+}
+
+// function to add data on local storage
+function addDataToLocal(arrayOfTasks) {
+  window.localStorage.setItem('tasks', JSON.stringify(arrayOfTasks));
+}
 
-displayTasks(lists);
+function getDataFromLocal() {
+  const data = window.localStorage.getItem('tasks');
+  if (data) {
+    const tasks = JSON.parse(data); // convert into an object
+    addElementsToPageFrom(tasks);
+  }
+}
+
+function deleteTaskWith(taskId) {
+  // filter //taskId = e.target.parentelemnt.getAttribute(data-id);
+  // eslint-disable-next-line eqeqeq
+  arrayOfTasks = arrayOfTasks.filter((task) => task.id != taskId);
+  addDataToLocal(arrayOfTasks);
+}
+
+function toggleStatusTask(taskId) {
+  for (let i = 0; i < arrayOfTasks.length; i += 1) {
+    // eslint-disable-next-line eqeqeq
+    if (arrayOfTasks[i].id == taskId) {
+      // eslint-disable-next-line no-unused-expressions
+      arrayOfTasks[i].completed == false
+        ? (arrayOfTasks[i].completed = true)
+        : (arrayOfTasks[i].completed = false);
+    }
+  }
+  addDataToLocal(arrayOfTasks);
+}
+
+deletebtn.addEventListener('click', () => {
+  // tasksDiv.innerHTML = " ";
+  // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ] Please don't disable linter rules. These are here to write clean and standard code.